src/community.rs

```diff
@@ -0,0 +1,54 @@
+use near_sdk::borsh::{self, BorshDeserialize, BorshSerialize};
+use near_sdk::serde::{Deserialize, Serialize};
+use near_sdk::{env, AccountId};
+
+#[derive(BorshSerialize, BorshDeserialize, Serialize, Deserialize, Clone)]
+#[serde(crate = "near_sdk::serde")]
+pub struct Github {
+    pub repo: String,
+    pub labels: Vec<String>,
+}
+
+#[derive(BorshSerialize, BorshDeserialize, Serialize, Deserialize, Clone)]
+#[serde(crate = "near_sdk::serde")]
+pub struct Community {
+    pub name: String,
+    pub description: String,
+    pub image: String,
+    pub thumbnail: String,
```

Comment:
Are these IPFS ids or full URLs? Please, rename the fields to provide this extra context. Consider using a dedicated type to validate the value.

Reply:
OK